src/main/java/com/aliyuncs/fc/model/TimeTriggerConfig.java

```diff
@@ -0,0 +1,48 @@
+package com.aliyuncs.fc.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Time Trigger config
+ */
+public class TimeTriggerConfig {
```

Comment:
Can you follow log trigger configs' convention? e.g. https://github.com/aliyun/fc-java-sdk/blob/aebb97aa8cbefc28c622b1f8bda876c9cdcad6d7/src/main/java/com/aliyuncs/fc/model/LogTriggerConfig.java

if I just want to disable trigger, I don't need to set `cronExpression` and `payload` as below, I can just do `new TimeTriggerConfig().SetEnable(false)`.

`updateReq.setTriggerConfig(new TimeTriggerConfig(cronExpression, payload,true));`